feature: add Dutch to list of languages

jlengrand opened the pull request:
Hey again @Janiczek , 

I also happen to speak Dutch :).

This MR makes more sense, because Dutch has a different default case than French and English :).

Just as for French, @swendude and I offer to help growing as the repository gets more complete.

Cheers,

Janiczek commented on src/PluralRules/Nl.elm:
```diff
@@ -0,0 +1,90 @@
+    module Pluralize exposing (pluralize)
+
+    import PluralRules exposing (Cardinal(..), Rules)
+    import PluralRules.En
```
I wonder if we want to have such example in each of the language modules. Currently the English one has it but the Czech one doesn't.

Do you think it would be good to have it everywhere? Or should we just point people to `examples/`? Or something else?

In any case, if the examples are here to stay, perhaps let's change the code example to use `import PluralRules.Nl`, `PluralRules.Nl.pluralize` and some irregular (non-`-en`) Dutch word in the `rules` dictionary :slightly_smiling_face:

jlengrand replied:
I think the doc should be as barebone as possible for most languages. You only need english (very simple) , and Czesh (much complication :D) and you can only refer to them as better examples. We can add more doc later, should it be needed.

Janiczek commented on tests/PluralRulesTest.elm:
```diff
@@ -54,6 +55,38 @@ fromFloatInt =
 
 
+rulesWithKind : Rules
+rulesWithKind =
```
I wonder if sometime we'll need to somehow prefix these with the language they're for :joy: Today is not that day, but still...

Janiczek commented:
Let's figure out the fate of the example in doc-comment, and this is good to go! :tada:

Janiczek commented on src/PluralRules/Nl.elm:
```diff
@@ -0,0 +1,90 @@
+
+# Usage
+
+Create your own dictionary of "non-s" pluralizations and a helper function that
```
btw `non-s` :arrow_right: `non-en`?

jlengrand commented:
Alright, so I updated to be like I would do it in my repo : Reduce duplication of the doc, and point to the most simple one (english), and the most complex one (Czech). 

I also tried to use the En implementation to reduce code duplication but the defaultPluralize setting gets in the way. It might be worth lifting it in case this library gets traction to avoid copy paste for similar languages. 

I also added a list of supported languages in the README, since I'm adding 2 more :).

Janiczek commented:
Looks great! (I think I wouldn't like reusing rules between languages :shrug: so I'm glad it ended up how it ended up :smile: )
